transport-data/tools
# Create IAMC ‘variable’ codelists

This PR adds a utility function that creates an SDMX Codelist for the ‘VARIABLE’ dimension of an IAMC-like structure from an existing data structure definition (DSD).

## How to review
- View the auto-generated docs build for the code on this branch: https://transport-data--10.org.readthedocs.build/en/10/iamc.html
  - Ensure the added code is described clearly.
